loader/src/main/java/net/neoforged/fml/loading/LoadingModList.java

```diff
@@ -87,6 +93,22 @@ public void addAccessTransformers() {
                 .forEach(mod -> mod.getAccessTransformers().forEach(path -> FMLLoader.addAccessTransformer(path, mod)));
     }
 
+    public void addEnumExtenders() {
+        Map<IModInfo, Path> pathPerMod = new HashMap<>();
```

Comment:
Is there value in allowing a mod to have multiple extenders?

Reply:
Not sure, to be honest